### DISCUSSION TITLE
recipes/ingredients/caramel/dry-caramel/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1243] [Comments]
# recipes/ingredients/caramel/dry-caramel/

A collection of my recipes.

https://nicholaswilde.io/recipes/ingredients/caramel/dry-caramel/

**Comment by nicholaswilde** [comment]
- https://food52.com/recipes/86558-salted-vanilla-bean-caramels-recipe
- https://youtu.be/Ip6uLO4yVkA?si=0Mu4KCnZBgiEAdjC
